packages/@aws-cdk/aws-appmesh/lib/virtual-gateway.ts

```diff
@@ -186,6 +199,13 @@ export class VirtualGateway extends VirtualGatewayBase {
       resourceName: this.physicalName,
     });
   }
+
+  /**
+   * Adds Default Backend Configuration for virtual node to communicate with Virtual Services.
+   */
+  private addBackendDefaults(backendDefaults: ClientPolicy) {
```

Comment:
I would get rid of this method, and turn the `backendDefaults` field to a local variable.

Also, there can only be a single `ClientPolicy` passed in `backendDefaults`. Why, since the L1 class takes in an array?

Reply:
Updated and created a local constructor variable. The backend default would accepts an object instead of an array.